### DISCUSSION TITLE
[M-Z] (10 Febbraio 2020) Esercizio 1

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#458] [Esami & Esoneri M-Z]
## Traccia

<img width="455" alt="image" src="https://github.com/user-attachments/assets/2cfbf6e0-99e8-4676-8e19-bbfdefbe77ec">

**Comment by rimaout** [comment]
<img width="600" alt="image" src="https://github.com/user-attachments/assets/be5f3a26-3945-4065-a9ca-5b6e5a635e38">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20200210.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20200210.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/d2a4e9b5-6c1c-4ad5-9117-cba2059ef4f4)
